src/encoded/static/components/regulome_search.js

```diff
@@ -290,8 +292,8 @@ class AdvSearch extends React.Component {
                 {(context.notification && context.notification !== 'No annotations found') ?
                     <div className="notification">{context.notification}</div>
                 : null}
-                {(context.variants && context.variants.length > 0) ?
-                    <p>Searched coordinates: {Object.keys(context.variants)[0]}</p>
+                {(context.query_coordinates && context.query_coordinates.length > 0) ?
```

Comment:
Have you considered saving context.query_coordinates in a variable?

Reply:
No. Maybe there is an advantage? Again I was trying to mimic the behavior of `context.variants`.